### PR TITLE
First pass at library, stacked recording cards

### PR DESCRIPTION
**Pull request by mattleff**
Fixes https://trello.com/c/hQR96Jiu/4-recording-detail, refs a bunch...

localhost_3000_en_library (1)

### CODE REVIEW

File: src/components/molecules/card/favorite.graphql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+fragment cardFavorite on UserFavorite {
+	createdAt
+	entity {
+		__typename
+		... on Recording {
```
Comment (narthur) [inline src/components/molecules/card/favorite.graphql:5]: Is this syntax a switch statement? Pretty slick...

File: src/components/organisms/navigation.tsx [hunk 2]
```diff
@@ -41,6 +41,7 @@ const Navigation = ({ onExit }: { onExit: () => void }): JSX.Element => {
 		},
 		{
 			key: 'playlist',
+			href: `/${languageRoute}/library`,
```
Comment (narthur) [inline src/components/organisms/navigation.tsx:44]: Should this use a route function?

File: src/components/organisms/libraryNav.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+						key={href}
+						className={clsx(href === currentNavHref && styles.miniNavActive)}
+					>
+						<Link href={href}>
+							<a>{label}</a>
+						</Link>
+					</Heading6>
+				))}
+			</div>
+			<Button
```
Comment (narthur) [inline src/components/organisms/libraryNav.tsx:73]: Do we need a todo for wiring up this sort button?

**Review by narthur — APPROVED** [review]
great work!

File: src/components/organisms/libraryNav.tsx (hunk 3 above)
Comment (mattleff) [reply src/components/organisms/libraryNav.tsx:74]: Added: 4a74bcf

File: src/components/organisms/navigation.tsx (hunk 2 above)
Comment (mattleff) [reply src/components/organisms/navigation.tsx]: Added: 4a74bcf

File: src/containers/account/playlists.tsx [hunk 4]
```diff
@@ -112,11 +112,11 @@ function Playlists(): JSX.Element {
 					checked={isPublic}
 					setChecked={setIsPublic}
 				/>
-				<button type={'submit'}>
+				<button type="submit">
```
[the remark this answers is not in the document]
Comment (mattleff) [reply src/containers/account/playlists.tsx:115]: Turns out we can: 4a74bcf ✨

File: src/components/molecules/card/base/withTheme.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+	| 'sermon'
+	| 'song'
+	| 'story'
+	| 'topic';
+
+interface CardProps {
+	theme: CardTheme;
+}
+
+export interface ICardHat {
```
[the remark this answers is not in the document]
Comment (mattleff) [reply src/components/molecules/card/base/withTheme.tsx]: Yeah, this isn't needed at all: 4a74bcf

File: src/components/molecules/card/favorite.graphql (hunk 1 above)
Comment (mattleff) [reply src/components/molecules/card/favorite.graphql:5]: The `entity` field is a GraphQL Union, so this syntax is telling GraphQL what properties from each type I want.

**Pull request merged**